pkg/api/api.go

```diff
@@ -33,9 +33,12 @@ type PrepareForSubmarinerInput struct {
 
 // Cloud is a potential cloud for installing Submariner on.
 type Cloud interface {
-	// PrepareForSubmariner will prepare the cloud for Submariner to operate on.
+	// Deprecated: Use OpenPorts instead, this will be removed on the 0.16 release.
 	PrepareForSubmariner(input PrepareForSubmarinerInput, status reporter.Interface) error
 
+	// OpenPorts inside the cloud for submariner to communicate through.
+	OpenPorts(ports []PortSpec, status reporter.Interface) error
+
 	// CleanupAfterSubmariner will clean up the cloud after Submariner is removed.
 	CleanupAfterSubmariner(status reporter.Interface) error
```

Comment:
Should this be replaced by a `ClosePorts` while we’re at it?

Reply:
I was thinking of it, could do if the direction seems good